paasta_tools/setup_istio_mesh.py

```diff
@@ -172,6 +172,37 @@ def setup_paasta_namespace_services(
     return status
 
 
+def cleanup_kube_svc(
+    kube_client: KubeClient,
+    smartstack_namespaces: AbstractSet,
+    existing_svc_names: Set[str],
+) -> bool:
+    sanitized_smarstack_namespaces = {
+        sanitise_kubernetes_service_name(namespace)
+        for namespace in smartstack_namespaces
+    }
+
+    status = True
+    if not existing_svc_names:
+        return status
+
+    existing_svc_names = existing_svc_names.difference(sanitized_smarstack_namespaces)
+
+    for svc in existing_svc_names:
+        if svc != UNIFIED_K8S_SVC_NAME:
+            try:
+                kube_client.core.delete_namespaced_service(
+                    name=svc, namespace=PAASTA_NAMESPACE
+                )
+                log.debug(
```

Comment:
I think it's probably useful to add this as a `log.info` since we'll want to see which svc objects are being deleted by the script.

please also add this above the delete call above, so we can see the intent even if it fails